sdk/storage/azure-storage-files-datalake/sample/datalake_getting_started.cpp

```diff
@@ -24,7 +24,7 @@ void DataLakeGettingStarted()
   // Initializing a FileSystemClient that can then initialize the PathClient, FileClient,
   // DirectoryClient.
   auto fileSystemClient
-      = FileSystemClient::CreateFromConnectionString(GetConnectionString(), fileSystemName);
+      = DataLakeFileSystemClient::CreateFromConnectionString(GetConnectionString(), fileSystemName);
```

Comment:
Not for this PR, but why not `serviceClient.GetFileSystemClient(fileSystemName)`?

Reply:
I don't think using either constructor or static create method is a must. I do believe it is an improvement though, just not major.